plugins/devtools/src/api/DevToolsClient.ts

```diff
@@ -56,16 +57,28 @@ export class DevToolsClient implements DevToolsApi {
   public async getInfo(): Promise<DevToolsInfo | undefined> {
     const urlSegment = 'info';
 
-    const info = await this.get<DevToolsInfo | undefined>(urlSegment);
+    const info = await this.request<DevToolsInfo | undefined>(urlSegment);
     return info;
   }
 
-  private async get<T>(path: string): Promise<T> {
+  public async getTasks(): Promise<TaskInfo[] | undefined> {
+    const urlSegment = 'tasks';
+    const tasks = await this.request<TaskInfo[] | undefined>(urlSegment);
+    return tasks;
+  }
+
+  public async triggerTask(scheduler: string, task: string): Promise<void> {
+    const urlSegment = `tasks/${scheduler}/${task}`;
+    await this.request(urlSegment, 'POST');
+  }
+
+  private async request<T>(path: string, method?: string): Promise<T> {
     const baseUrl = `${await this.discoveryApi.getBaseUrl('devtools')}/`;
     const url = new URL(path, baseUrl);
 
     const { token } = await this.identityApi.getCredentials();
     const response = await fetch(url.toString(), {
+      method,
```

Comment:
I like this change, just curious if you don't pass in `method` it would be then be `undefined` and `fetch` doesn't have any issues with that so defaults to a `GET`?

Reply:
Yes exactly, fetch defaults to GET if the method is not defined

Reply:
OK, cool, thanks!